client/css/style.css

```diff
@@ -1948,7 +1949,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 }
 
 .topic .emoji {
-	vertical-align: middle;
+	vertical-align: text-bottom;
```

Comment:
I would prefer if we leave this at `middle` and actually change `.topic .emoji` into `.emoji`.
(I just noticed that emoji in messages are aligned with `bottom` except in the "The topic is:" action messages where this rule takes over `.emoji`).

Reply:
So this either depends on font or something else, because on my end  `middle` looks too low and `text-bottom` makes it more centered.

Reply:
That's weird!

Reply:
To be fair I expect middle to be vertically centered, it feels odd that text-bottom centers.